sonar-project.properties

```diff
@@ -0,0 +1,36 @@
+# Organization and project keys are displayed in the right sidebar of the project homepage
+sonar.organization=owncloud-1
+sonar.projectKey=owncloud_core
+sonar.projectVersion=1.0
+sonar.host.url=https://sonarcloud.io
+
+# =====================================================
+#   Meta-data for the project
+# =====================================================
+
+sonar.links.homepage=https://github.com/owncloud/core
+sonar.links.ci=https://drone.owncloud.com/owncloud/core/
+sonar.links.scm=https://github.com/owncloud/core
+sonar.links.issue=https://github.com/owncloud/core/issues
+
+# =====================================================
+#   Properties that will be shared amongst all modules
+# =====================================================
+
+# Just look in these directories for code
+sonar.sources=.
+sonar.inclusions=apps/**,core/**,lib/**,ocs/**,settings/**
+
+# Pull Requests
+sonar.pullrequest.provider=GitHub
+sonar.pullrequest.github.repository=owncloud/core
+sonar.pullrequest.base=${env.SONAR_PULL_REQUEST_BASE}
+sonar.pullrequest.branch=${env.SONAR_PULL_REQUEST_BRANCH}
+sonar.pullrequest.key=${env.SONAR_PULL_REQUEST_KEY}
+
+# Properties specific to language plugins:
+sonar.php.coverage.reportPaths=results/clover-phpunit-php7.2-mariadb10.2.xml,results/clover-phpunit-php7.2-mariadb10.3.xml,results/clover-phpunit-php7.2-mariadb10.4.xml,results/clover-phpunit-php7.2-mariadb10.5.xml,results/clover-phpunit-php7.2-mysql5.5.xml,results/clover-phpunit-php7.2-mysql5.7.xml,results/clover-phpunit-php7.2-mysql8.0.xml,results/clover-phpunit-php7.2-oracle.xml,results/clover-phpunit-php7.2-postgres10.3.xml,results/clover-phpunit-php7.2-postgres9.4.xml,results/clover-phpunit-php7.2-sqlite-samba-samba.xml,results/clover-phpunit-php7.2-sqlite-samba.xml,results/clover-phpunit-php7.2-sqlite-scality.xml,results/clover-phpunit-php7.2-sqlite-sftp-sftp.xml,results/clover-phpunit-php7.2-sqlite-sftp.xml,results/clover-phpunit-php7.2-sqlite-webdav-webdav.xml,results/clover-phpunit-php7.2-sqlite-webdav.xml,results/clover-phpunit-php7.2-sqlite-windows-windows.xml,results/clover-phpunit-php7.2-sqlite-windows.xml,results/clover-phpunit-php7.2-sqlite.xml,results/clover-phpunit-php7.3-mariadb10.2.xml,results/clover-phpunit-php7.3-sqlite.xml,results/clover-phpunit-php7.4-mariadb10.2.xml,results/clover-phpunit-php7.4-sqlite.xml
+sonar.javascript.lcov.reportPaths=results/lcov.info
+
+# Exclude translation, dependency, 3rd-party and test files
+sonar.exclusions=build/**,changelog/**,config/**,core/l10n/**,core/vendor/**,l10n/**,lib/l10n/**,lib/composer/**,settings/l10n/**,resources/**,settings/tests/**,tests/**,vendor-bin/**,apps/files_external/3rdparty/**,apps/**/l10n/**,apps/**/tests/**,apps/**/vendor/**
```

Comment:
I think this setting is only needed if we want to exclude some things that are inside `sonar.inclusions`
For this app it looks like the whole of `sonar.inclusions` can be analysed - so delete this line.

The `sonar.inclusions` and `sonar.exclusions` might vary a bit for each app (e.g. when doing ones that have translations or "odd" 3rd-party code)

Reply:
done